Documentation/Cookbooks/translations.md

```diff
@@ -1,3 +1,260 @@
-Model vs Entities translation
-static translations
-context translations language/region
+Internationalization
+====================
+
+How to translate your project ?
+
+> Note: Internationalization and Contextualization are two differents things. 
+>
+> Internationalization is used for translating UI while Contextualization is used when storing data in different languages.
+> 
+> For example, someone uses your app in english (internationalization), while storing a product description in french (Contextualization).
+>
+> Read more about data contextualization on [the official documentation](https://vincentchalnot.github.io/SidusEAVModelBundle/Documentation/09-context.html)
+ 
+# Translation files
+
+This project uses Symfony translation mechanics. You can read [the official documentation](https://symfony.com/doc/3.4/translation.html) first. 
+
+By default, the langage used is based on your project configuration, inside `app/config/config.yml`
+
+```yaml
+parameters:
+    locale: fr
+````
+
+If you want you app to be available in multiple langagues, you'll have to use Symfony `{_locale}` parameter in your routes. Follow [the official routing documentation](https://symfony.com/doc/3.4/routing.html) to learn more.
+
+# Translations
+
+## Translating family name
+
+Here is an example of a simple family translation
+
```

Comment:
Add a short phrase before the model example: "Let's say your model looks like this:" or "Given this model configuration:"